apps/webClient/src/components/View/index.tsx

```diff
@@ -1,54 +1,90 @@
 import React from "react";
+import { Link } from "react-router-dom";
+import clsx from "clsx";
+import { useLocation } from "react-router-dom";
 import { makeStyles } from "@material-ui/styles";
 import Container from "@material-ui/core/Container";
-import Box from "@material-ui/core/Box";
+import Grid from "@material-ui/core/Grid";
+import Logo from "../Logo";
 
 interface Props {
   children: React.ReactNode;
 }
 
-// placholder, should dynamically change via useLocation
-const LIGHT_BLUE = "rgb(177,32,41)";
-const GRADIENT =
+const RED = "rgb(177,32,41)";
+const RED_GRADIENT =
   "linear-gradient(45deg, rgba(177,32,41,1) 0%, rgba(244,121,32,1) 100%)";
+const GREEN = "rgb(14,124,63)";
+const GREEN_GRADIENT =
+  "linear-gradient(45deg, rgba(14,124,63,1) 0%, rgba(190,198,64,1) 100%)";
+
+const BLUE = "rgb(28,85,119)";
+const BLUE_GRADIENT =
+  "linear-gradient(45deg, rgba(28,85,119,1) 0%, rgba(46,192,209,1) 100%)";
+const YELLOW = "rgb(190,121,42)";
+const YELLOW_GRADIENT =
+  "linear-gradient(45deg, rgba(190,121,42,1) 0%, rgba(254,228,104,1) 100%)";
 
 const useStyles = makeStyles(theme => ({
   root: {
     width: "100vw",
-    height: "100vh",
+    height: "100%",
+    minHeight: "100vh",
     zIndex: 1,
     overflow: "hidden",
     display: "flex",
     flexDirection: "column",
   },
   main: {
     width: "100%",
+    height: "100%",
     flexGrow: 1,
-    overflowY: "auto",
-    overflowX: "scroll",
+    padding: 0,
+    marginTop: 128,
   },
-  pageWrapper: {
-    backgroundColor: LIGHT_BLUE,
-    background: GRADIENT,
-    height: "100%",
+  redPageWrapper: {
+    backgroundColor: RED,
+    background: RED_GRADIENT,
   },
-  pageContainer: {
-    padding: 0,
+  greenPageWrapper: {
+    backgroundColor: GREEN,
+    background: GREEN_GRADIENT,
+  },
+  bluePageWrapper: {
+    backgroundColor: BLUE,
+    background: BLUE_GRADIENT,
+  },
+  yellowPageWrapper: {
+    backgroundColor: YELLOW,
+    background: YELLOW_GRADIENT,
   },
 }));
 
 const View: React.FC<Props> = ({ children }) => {
   const classes = useStyles();
+  const { pathname } = useLocation();
+
+  const derivePageRootBackgroundColor = (path: string) => {
+    let coloredPageRoot = [classes.root];
+    if (path === "/") coloredPageRoot.push(classes.redPageWrapper);
+    if (path === "/our-work") coloredPageRoot.push(classes.greenPageWrapper);
+    if (path === "/about") coloredPageRoot.push(classes.bluePageWrapper);
+    if (path === "/lets-talk") coloredPageRoot.push(classes.yellowPageWrapper);
```

Comment:
No need to change this solution, but `switch` is the current best practice here.

Optimization people will say that each `if` statement is evaluated and slower than using if/else or `switch`